tools/clang/test/HLSLFileCheck/passes/hl/sroa_hlsl/NodeInput_type.ll

```diff
@@ -8,6 +8,22 @@
 ; CHECK: alloca %"struct.DispatchNodeInputRecord<loadStressRecord>"
 ; CHECK: load %"struct.DispatchNodeInputRecord<loadStressRecord>", %"struct.DispatchNodeInputRecord<loadStressRecord>"*
 
+; Buffer Definitions:
+;
+; cbuffer $Globals
+; {
+;
+;   [0 x i8] (type annotation not present)
+;
+; }
+;
+;
+; Resource Bindings:
+;
+; Name                                 Type  Format         Dim      ID      HLSL Bind  Count
+; ------------------------------ ---------- ------- ----------- ------- -------------- ------
+; $Globals                          cbuffer      NA          NA     CB0   cb4294967295     1
+;
```

Comment:
There's at least some disagreement about whether this commented information is useful in a committed .ll test case. It's not strictly necessary.